pack/README.md

```diff
@@ -0,0 +1,40 @@
+# Use Pack to build container images
+
+Build container images using [pack](https://buildpacks.io/docs/install-pack/) and [buildpacks](https://buildpacks.io/).
+
+## Usage
+
+This extension adds the `pack` function, which is used the same way as `docker_build` to allow Tilt
+to automatically build a container image with a known name. For instance the following example shows
+building an image using the default builder and having that image automatically deployed to Kubernetes.
+
+```python
+load('ext://pack', 'pack')
+
+pack('example-image')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-deployment', port_forwards=8000)
+```
+
+The `pack` function takes two new arguments:
```

Comment:
"new" seems wrong. "optional"?

It'd probably also be good to mention `name` here. Reading this before the code, I was unaware it existed and thought `"example-image"` above was being passed as `path`.

Reply:
Thanks. Fixed up the language and added the `name` argument as well.